python/ray/serve/api.py

```diff
@@ -83,6 +83,13 @@ def shutdown(self) -> None:
             ray.kill(self._controller, no_restart=True)
             self._shutdown = True
 
+    @_ensure_connected
+    def get_result(self, result_object_id: ray.ObjectRef) -> FutureResult:
```

Comment:
Can we document this API and/or make it private?

Reply:
should be private

Reply:
SGTM!